src/components/artist/ArtistCard.js

```diff
@@ -9,9 +9,9 @@ import {
 } from 'common/hooks/useImageSize'
 import { WidthSizes, SquareSizes } from 'common/models/ImageSizes'
 import DynamicImage from 'components/dynamic-image/DynamicImage'
-import FollowButton from 'components/general/FollowButton'
-import Stats from 'components/general/Stats'
-import UserBadges from 'containers/user-badges/UserBadges'
+import FollowButton from 'components/follow-button/FollowButton'
```

Comment:
just a thought for the future, how would we feel about an import style like
```
import {FollowButton} from 'components/follow-button'
```
so essentially follow-button directory has an index that exports the api for this component/feature?

Reply:
I'm a fan of that, we're doing it in the mobile client

Reply:
https://github.com/AudiusProject/protocol-dashboard/tree/main/src/components/AppBar we do it in the protocol dashboard as well

Reply:
one thing at a time though 😓